bank_rec/src/index.css

```
@@ -2,6 +2,199 @@
 @tailwind components;
 @tailwind utilities;
 
+:root {
+  color-scheme: light;
+  --font-stack: "InterVariable", "Inter", -apple-system, BlinkMacSystemFont,
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)